utils/writeable/README.md

```diff
@@ -43,6 +39,10 @@ impl<'s> Writeable for WelcomeMessage<'s> {
 
 let message = WelcomeMessage { name: "Alice" };
 assert_writeable_eq!(&message, "Hello, Alice!");
+
+// Types implementing `Writeable` are required to also implement `fmt::Display`.
+// This can be simply done by redirecting to the `Writeable` implementation:
```

Comment:
```suggestion
// Types implementing `Writeable` should often also implement `fmt::Display`.
// This can be simply done by redirecting to the `Writeable` implementation:
```